### PR TITLE
Release section for 1.10

### PR DESCRIPTION
**Pull request by judithpatudith**
## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-website#testing-your-updates-locally) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-website#contribution-workflow).

### CODE REVIEW

**Comment by judithpatudith** [comment]
Hey, team, I got approval for the text from prod and marketing but would love two really thorough reviews before merging. Submitting a separate PR to pin the 1.10.0-rc1 release notes download button to a specific version. Thank you!

**Comment by judithpatudith** [comment]
Hi, I'd love a review of this before we ship any more 1.10 versions in order to avoid merge conflicts. @karlkfi and @joerg84 or @pleia2 do you have time tomorrow to review it? Thanks!

File: src/releases.json [hunk 1]
```diff
@@ -1,5 +1,30 @@
 [
   {
+    "title": "DC/OS 1.10 (Beta)",
+    "date": "2017-08-02",
+    "description": "DC/OS 1.10 includes core DC/OS service improvements, CNI networking enhancements, and data service enhancements.",
+    "features": [
+      "User Interface: Streamlined GUI, multi-cluster CLI support.",
+      "Container Orchestration: UCR improvements.",
+      "Platform: Node health checks, enhanced upgrades with pre/post flight checks.",
+      "Networking: Increased CNI support, configurable Spartan upstreams for domains.",
```
Comment (joerg84) [inline src/releases.json]: S/increased/improved

Comment (joerg84) [reply src/releases.json]: s/upstreams/upstream DNS server

File: src/releases.json [hunk 2]
```diff
@@ -1,5 +1,30 @@
...
+      "Data Services: Deployment into folders, deployment to CNI-based virtual networks.",
+      "Apache Mesos 1.4.",
+      "Marathon 1.5."
+    ],
+    "downloadlink": {
+      "amazon": "https://downloads.dcos.io/dcos/EarlyAccess/aws.html",
+      "azure": "https://downloads.dcos.io/dcos/EarlyAccess/azure.html",
+      "direct": "https://downloads.dcos.io/dcos/EarlyAccess/commit/a5ecc9af5d9ca903f53fa16f6f0ebd597095652e/dcos_generate_config.sh"
```
Comment (joerg84) [inline src/releases.json]: Do we have to use the commit here?

File: src/releases.json [hunk 3]
```diff
@@ -1,5 +1,30 @@
...
+      "User Interface: Streamlined GUI, multi-cluster CLI support.",
+      "Container Orchestration: UCR improvements.",
+      "Platform: Node health checks, enhanced upgrades with pre/post flight checks.",
+      "Networking: Increased CNI support, configurable Spartan upstreams for domains.",
+      "Data Services: Deployment into folders, deployment to CNI-based virtual networks.",
+      "Apache Mesos 1.4.",
+      "Marathon 1.5."
+    ],
+    "downloadlink": {
+      "amazon": "https://downloads.dcos.io/dcos/EarlyAccess/aws.html",
```
Comment (joerg84) [inline src/releases.json]: If it is released the link should be to stable not EA or?

Comment (judithpatudith) [reply src/releases.json]: Currently, it should be EA (if we merge this before releasing GA)

Comment (joerg84) [reply src/releases.json]: Thx

**Pull request merged**